pkg/datapath/iptables/iptables.go

```diff
@@ -1351,28 +1351,29 @@ func (m *IptablesManager) addCiliumNoTrackXfrmRules() error {
 	return nil
 }
 
-func (m *IptablesManager) addNoTrackPodToPodRules(prog, podsCIDR string) error {
-	if err := runProg(prog, append(
-		m.waitArgs,
-		"-t", "raw",
-		"-I", ciliumPreRawChain,
-		"-s", podsCIDR,
-		"-d", podsCIDR,
-		"-m", "comment", "--comment", "cilium: NOTRACK for pod-to-pod traffic",
-		"-j", "NOTRACK"),
-		false); err != nil {
-		return err
-	}
-	if err := runProg(prog, append(
-		m.waitArgs,
-		"-t", "raw",
-		"-I", ciliumOutputRawChain,
-		"-s", podsCIDR,
-		"-d", podsCIDR,
-		"-m", "comment", "--comment", "cilium: NOTRACK for pod-to-pod traffic",
-		"-j", "NOTRACK"),
-		false); err != nil {
-		return err
+func (m *IptablesManager) addNoTrackPodTrafficRules(prog, podsCIDR string) error {
+	for _, chain := range []string{ciliumPreRawChain, ciliumOutputRawChain} {
+		if err := runProg(prog, append(
+			m.waitArgs,
+			"-t", "raw",
+			"-I", chain,
+			"-s", podsCIDR,
+			"-m", "comment", "--comment", "cilium: NOTRACK for pod traffic",
+			"-j", "NOTRACK"),
+			false); err != nil {
+			return err
+		}
+
+		if err := runProg(prog, append(
```

Comment:
Isn't it exactly the same `iptables` command as the one above? I see no difference and seems like it's a dup. ;)

Reply:
They are actually 2 different rules:

* first one has `"-s", podsCIDR`
* second one has `"-d", podsCIDR`

Reply:
OK, sorry then!

Reply:
No worries :raised_hands: